custom_components/ferroamp/config_flow.py

```diff
@@ -56,13 +66,61 @@ async def async_step_init(self, user_input=None):
         if interval is None or interval == 0:
             interval = 30
 
+        precision_battery = self.config_entry.options.get(CONF_PRECISION_BATTERY)
+        if precision_battery is None:
+            precision_battery = 1
+
+        precision_current = self.config_entry.options.get(CONF_PRECISION_CURRENT)
+        if precision_current is None:
+            precision_current = 0
+
+        precision_energy = self.config_entry.options.get(CONF_PRECISION_ENERGY)
+        if precision_energy is None:
+            precision_energy = 1
+
+        precision_power = self.config_entry.options.get(CONF_PRECISION_POWER)
+        if precision_power is None:
+            precision_power = 0
```

Comment:
This (precision=0) will result in power for SSO which now is in kW will only display 1 kW if eg 1.34 kW, right? I think that's too inexact as default for raw data like this. Default should be 2 if we want to keep it as kW.

Reply:
True. Changing. Agree on leaving formatting to presentation layer.

Reply:
Hmmm, I use the power precision in the `ThreePhasePowerFerroampSensor` and there are decimals there. Not sure they are needed ever though. Thoughts @henricm ?

Reply:
Removed power precision completely